kubernetes/structures_pod.go

```diff
@@ -439,6 +487,24 @@ func expandPodSpec(p []interface{}) (v1.PodSpec, error) {
 		obj.TerminationGracePeriodSeconds = ptrToInt64(int64(v))
 	}
 
+	if v, ok := in["toleration"].([]map[string]interface{}); ok && len(v) > 0 {
+		tolerations := make([]v1.Toleration, len(v))
+		for i, t := range v {
+			tol := v1.Toleration{
+				Key:      t["key"].(string),
+				Operator: tolerationOperator(t["operator"].(string)),
```

Comment:
Hmm, I added a comment here earlier but it got lost.
This could be simplified to:
`Operator: v1.TolerationOperator(t["operator"].(string)),`

For the same reasons as the other comments.